internal/api/handler/achievement_handler.go

```diff
@@ -40,3 +41,25 @@ func CreateAchievementHandler(ctx context.Context, profileSvc service.Service) f
 		})
 	}
 }
+
+func GetAchievementsHandler(ctx context.Context, achSvc service.Service) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		profileID, err := helpers.GetParams(r)
```

Comment:
Can we define new function like `GetParamByID("profile_id")` and get profileID

Reply:
GetParam is already defined for the getting the profileID so can i rename the function as GetParamByID

Reply:
lets discuss offline

Reply:
kk